crates/uv-platform-tags/src/abi_tag.rs

```diff
@@ -265,6 +267,15 @@ impl FromStr for AbiTag {
     }
 }
 
+impl FromStr for AbiTag {
+    type Err = ParseAbiTagError;
+
+    fn from_str(s: &str) -> Result<Self, Self::Err> {
+        // TODO(charlie): Gracefully handle unsupported tags.
```

Comment:
Is there a more graceful way we should be handling them? This sort of looks "handled".

Reply:
I think ideally we'd keep what we have today (so we have good error messages and keep the struct size small), but we'd filter out unknown tags when deserializing.

Reply:
What I have here is also acceptable though.